libs/wingsdk/src/target-sim/redis.inflight.ts

```diff
@@ -0,0 +1,79 @@
+import Dockerode from "dockerode";
+import { ISimulatorResourceInstance } from "./resource";
+import { RedisSchema } from "./schema-resources";
+import { IRedisClient } from "../redis";
+import { ISimulatorContext } from "../testing/simulator";
+
+// Issue using types from ioredis with JSII
+//import { Redis as IoRedis } from "ioredis";
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const IoRedis = require("ioredis");
+
+export class Redis implements IRedisClient, ISimulatorResourceInstance {
+  static uid = 0;
+  private readonly container_name: string;
+  private readonly context: ISimulatorContext;
+
+  private connection_url?: string = undefined;
+  private docker?: Dockerode = undefined;
+  private connection?: any;
+
+  public constructor(_props: RedisSchema["props"], context: ISimulatorContext) {
+    this.context = context;
+    this.container_name = `wing-sim-redis-${this.context.resourcePath.replace(
+      "/",
+      "."
+    )}-${Redis.uid}`;
+    Redis.uid++;
+  }
+
+  public async init(): Promise<void> {
+    this.docker = new Dockerode();
+
+    // Create a redis container
+    try {
+      let container = await this.docker.createContainer({
```

Comment:
Any reason to use `let`?

```suggestion
      const container = await this.docker.createContainer({
```